app/Http/Controllers/GameController.php

```diff
@@ -135,8 +138,13 @@ public function checkPassword(Game $game)
         return false;
     }
 
-    public function show(Game $game)
+    public function show($game_name)
```

Comment:
Je zit al in de `GameController`, is `name` ook niet duidelijk genoeg? Dat je gewoon `$name` ipv `$game_name` doet.